packages/mint-components/src/components/sqm-hero-image/sqm-hero-image.tsx

```diff
@@ -0,0 +1,122 @@
+import { withHooks } from "@saasquatch/stencil-hooks";
+import { Component, h, Prop } from "@stencil/core";
+import { Spacing } from "../../global/mixins";
+import { getProps } from "../../utils/utils";
+import { HeroImageView } from "./sqm-hero-image-view";
+
+/**
+ * @uiName Hero Image
+ */
+@Component({
+  tag: "sqm-hero-image",
+  shadow: true,
+})
+export class HeroImage {
+  /**
+   * @uiWidget ImageUpload
+   * @uiName Image Link
+   */
+  @Prop() imageUrl: string;
+
+  /**
+   * @uiName Overlay Color
+   * @uiWidget color
```

Comment:
This is an interesting default value for the prop to have with the color widget, as this is not actually a valid value for the color widget.

Reply:
Made this prop optional and added the shoelace value in the view as default